app/models/course.rb

```diff
@@ -62,7 +62,7 @@ def current_assessments(now = DateTime.now)
     assessments.where("start_at < :now AND end_at > :now", now: now)
   end
 
-  def display_name
+  def full_name
```

Comment:
Maybe check this with rubocop, but I think we should just refer to everything as an instance variable here (self[:semester] -> semester, etc).  I don't think we have code anywhere else that's doing anything like this.